packages/frontpage/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@atproto/oauth-types": "^0.1.2",
+    "@atproto/syntax": "^0.3.0",
```

Comment:
The `@atproto`ification of our code begins lol.

I'm gonna gradually convert our handling of core protocol things to the official libraries but definitely still don't wanna use the API clients because they're not really suited to our use case.